README.md

```diff
@@ -56,7 +56,11 @@
  ```
   /api/schema/
  ```
-
+## Questions Endpoints
+- User edit questio
```

Comment:
Poor attention to detail here. Please correct the typo here.

Reply:
Correcting it